### ISSUE TITLE
Unable to pull Renovate from DockerHub (failed to register layer)

### ISSUE THREAD

**Issue by promarcel** [renovatebot/docker-renovate-full#346]
### How are you running Renovate?

Self-hosted

### Please select which platform you are using if self-hosting.

GitLab self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

26, latest

### Describe the bug

Unfortunately, Since that night, I am unable to pull any image version of 26 Renovate from DockerHub.

I got the following error:

```
# docker image pull renovate/renovate:latest

latest: Pulling from renovate/renovate
16ec32c2132b: Already exists
d715208eb1c7: Already exists
765106c2e692: Already exists
0c0f3dd1a196: Already exists
c5c2857fb255: Already exists
3f567a8ca010: Already exists
42740552a891: Already exists
995f96f443d4: Already exists
472e3c8a8ad1: Already exists
aae510693a22: Already exists
5d1c05578460: Already exists
d46103cc2de7: Already exists
28035312d3b2: Pull complete
7fa3aea486f3: Pull complete
fa62b91b1679: Pull complete
0eb777e757d4: Pull complete
1c31e382eb4c: Pull complete
771cdfb10100: Pull complete
5add5deee88e: Extracting [==================================================>]  66.31MB/66.31MB
d99eb699437c: Download complete
7a5f9ec79c48: Download complete
bfce489f92eb: Download complete
e788d59b3f20: Download complete
185b10563062: Download complete
5c2ff99c4c1b: Download complete
b6e469fc88be: Download complete
434488d85c36: Download complete
107d9cec25da: Download complete
44ec1720dd19: Download complete
4146aa0efa97: Download complete
2d87596d6c0d: Download complete
63efb89cbbdc: Download complete
7090a12474a7: Download complete
751b00a4bdfe: Download complete
failed to register layer: ApplyLayer exit status 1 stdout:  stderr: lchown /usr/local/poetry/1.1.8/lib/poetry/_vendor/py2.7/backports/entry_points_selectable.py: invalid argument
```

It seems to have this issue right with this image because `/usr/local/poetry/1.1.8/lib/poetry/_vendor/py2.7/backports/entry_points_selectable.py` didn't exist on my local machine.

This error also occurs on other docker hosts, with the same message. Typical steps like restart the host, docker system prune or restart docker has already done without any issues.

Version 25 could be pulled without any issues, also any other images I use from DockerHub are working as expected.

### Relevant debug logs

<details><summary>Docker Version</summary>

```
# docker version

Client: Docker Engine - Community
 Version:           20.10.8
 API version:       1.41
 Go version:        go1.16.6
 Git commit:        3967b7d
 Built:             Fri Jul 30 19:54:27 2021
 OS/Arch:           linux/amd64
 Context:           default
 Experimental:      true

Server: Docker Engine - Community
 Engine:
  Version:          20.10.8
  API version:      1.41 (minimum version 1.12)
  Go version:       go1.16.6
  Git commit:       75249d8
  Built:            Fri Jul 30 19:52:33 2021
  OS/Arch:          linux/amd64
  Experimental:     false
 containerd:
  Version:          1.4.9
  GitCommit:        e25210fe30a0a703442421b0f60afac609f950a3
 runc:
  Version:          1.0.1
  GitCommit:        v1.0.1-0-g4144b63
 docker-init:
  Version:          0.19.0
  GitCommit:        de40ad0
```

</details>

<details><summary>Docker Info</summary>

```
# docker info

Client:
 Context:    default
 Debug Mode: false
 Plugins:
  app: Docker App (Docker Inc., v0.9.1-beta3)
  buildx: Build with BuildKit (Docker Inc., v0.6.1-docker)
  scan: Docker Scan (Docker Inc., v0.8.0)

Server:
 Containers: 39
  Running: 39
  Paused: 0
  Stopped: 0
 Images: 225
 Server Version: 20.10.8
 Storage Driver: overlay2
  Backing Filesystem: extfs
  Supports d_type: true
  Native Overlay Diff: false
  userxattr: false
 Logging Driver: fluentd
 Cgroup Driver: cgroupfs
 Cgroup Version: 1
 Plugins:
  Volume: local
  Network: bridge host ipvlan macvlan null overlay
  Log: awslogs fluentd gcplogs gelf journald json-file local logentries splunk syslog
 Swarm: inactive
 Runtimes: io.containerd.runc.v2 io.containerd.runtime.v1.linux runc
 Default Runtime: runc
 Init Binary: docker-init
 containerd version: e25210fe30a0a703442421b0f60afac609f950a3
 runc version: v1.0.1-0-g4144b63
 init version: de40ad0
 Security Options:
  apparmor
  seccomp
   Profile: default
 Kernel Version: 5.4.124-1-pve
 Operating System: Ubuntu 20.04.3 LTS
 OSType: linux
 Architecture: x86_64
 CPUs: 10
 Total Memory: 16GiB
 Name: [MASKED]
 ID: [MASKED]
 Docker Root Dir: /var/lib/docker
 Debug Mode: false
 Username: [MASKED]
 Registry: https://index.docker.io/v1/
 Labels:
 Experimental: false
 Insecure Registries:
  127.0.0.0/8
 Live Restore Enabled: false
```

</details>

### Have you created a minimal reproduction repository?

No reproduction repository

**Comment by viceice** [comment]
@promarcel checkout linked issue.

**Comment by viceice** [comment]
@promarcel containerbase/buildpack#124 should fix your issue

**Comment by viceice** [comment]
Duplicate of containerbase/buildpack#123

**Comment by viceice** [comment]
Closing this, when fix is bubbled through

**Comment by viceice** [comment]
fixed in 6f5ca19f8ea8f95495a9427d8bc03e21f1749d71

So when latest main actions are done, then all latest renovate full images are fixed.

**Comment by promarcel** [comment]
Thanks 🎉 It's working again!
